aws/cloudformation/components/session_store.yml.erb

```diff
@@ -15,11 +18,163 @@
       EngineVersion: 7.1
 
       ClusterMode: disabled
-      NumCacheClusters: 3 # 1 primary and 2 replicas
+<% num_cache_clusters = 3 # 1 primary and 2 replicas -%>
+      NumCacheClusters: <%= num_cache_clusters %>
       AutomaticFailoverEnabled: true
       MultiAZEnabled: true
       TransitEncryptionEnabled: true
       AtRestEncryptionEnabled: true
 
       SecurityGroupIds: [!ImportValue VPC-RedisSecurityGroup]
       CacheSubnetGroupName: !Ref SessionStoreSubnetGroup
+  # CloudWatch Alarms for the Redis-based Session Store
+  #
+  # Based on:
+  # https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/CacheMetrics.WhichShouldIMonitor.html
+  # and
+  # https://aws.amazon.com/blogs/database/monitoring-best-practices-with-amazon-elasticache-for-redis-using-amazon-cloudwatch/
+  SessionStoreCPUUsageAlarm:
+    Type: AWS::CloudWatch::Alarm
+    DependsOn: SessionStoreGroup
+    Properties:
+      AlarmName: <%="Redis Session Store CPU Usage Too High (#{stack_name})" %>
+      AlarmDescription: AWS Documentation recommends maintaining
+        EngineCPUUtilization below 90%. Note that we use EngineCPUUtilization
+        rather than CPUUtilization because the cache.r7g.xlarge nodes we're
+        using have 4vCPUs
```

Comment:
is this the correct multiline syntax? The lint or validate rake tasks should be sufficient to confirm.

Reply:
I verified with `lint`, `validate`, and also on an adhoc; see "Testing story" in the PR description